src/libs/common.go

```diff
@@ -458,7 +458,10 @@ func WriteCiliumPolicyToYamlFile(namespace string, policies []types.CiliumNetwor
 	}
 
 	for i := range policies {
-		jsonBytes := json.Marshal(&policies[i])
+		jsonBytes, err := json.Marshal(&policies[i])
+		if err != nil {
+			log.Error().Msg(err.Error())
```

Comment:
If json marshalling fails then we can't subsequently use the `jsonBytes`. In the error condition we should either `continue` or `break` as appropriate.